api/src/services/opportunities/opportunity_service_shared.py

```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
+
+from src.constants.lookup_constants import OpportunityCategory
+from src.db.models.opportunity_models import Opportunity
+from src.db.models.transfer.transfer_topportunity_models import TransferTopportunity
+
+
+def convert_transfer_opp_to_regular(transfer_opportunity: TransferTopportunity) -> Opportunity:
+    # Using transfer table directly from our opportunity endpoint is temporary
+    # until we implement the data transformation layer, this converts
+    # to what our API currently expects without requiring us to rewrite the Marshmallow/API layer
+
+    is_draft = None
+    if transfer_opportunity.is_draft == "Y":
+        is_draft = True
+    elif transfer_opportunity.is_draft == "N":
+        is_draft = False
+
+    opportunity_category = None
+    if transfer_opportunity.oppcategory:
+        opportunity_category = OpportunityCategory(transfer_opportunity.oppcategory)
+
+    # convert the update/create dates to datetime at 00:00:00
+    created_at, updated_at = None, None
+    if transfer_opportunity.created_date:
+        created_at = datetime.combine(transfer_opportunity.created_date, datetime.min.time())
```

Comment:
Do we need to think about timezone, e.g. something like this

```suggestion
        created_at = datetime.combine(
            transfer_opportunity.created_date,
            datetime.min.time(),
            tzinfo=zoneinfo.ZoneInfo("America/New_York"))
        )
```

Reply:
Agreed, we can figure this out later.